sysl2/sysl/validate/validate.go

```diff
@@ -113,18 +112,16 @@ func (v *Validator) validateFileName() {
 		v.messages[viewName] = append(v.messages[viewName], *msg.NewMsg(msg.ErrInvalidReturn, []string{viewName, "string"}))
 	}
 
-	assignCount := 0
-	for _, stmt := range view.GetExpr().GetTransform().GetStmt() {
+	for i, stmt := range view.GetExpr().GetTransform().GetStmt() {
 		if stmt.GetAssign() != nil {
-			if assignCount == 0 && stmt.GetAssign().GetName() != viewName {
+			if i == 0 && stmt.GetAssign().GetName() != viewName {
 				v.messages[viewName] = append(v.messages[viewName],
 					*msg.NewMsg(msg.ErrMissingReqField, []string{viewName, viewName, "string"}))
-			} else if assignCount > 0 {
+			} else if i > 0 {
```

Comment:
Switch the two tests around and remove `i == 0 && ` from the other one.

Reply:
Fixed